docs/resources/api_gateway_api.md

```diff
@@ -71,12 +71,12 @@ The following arguments are supported:
 
 * `http_backend` - (Optional, List) Specifies the configuration when backend_type selected 'HTTP' (documented below).
 
-* `function_backend` - (Optional, List) Specifies the configuration when backend_type selected 'FUNCTION' (documented
+* `function_backend` - (Required, List) Specifies the configuration when backend_type selected 'FUNCTION' (documented
```

Comment:
This field is Optional; please keep it as-is.

Reply:
fix